SECURITY.md

```diff
@@ -0,0 +1,28 @@
+# Security Policy
+
+## Supported Versions
+
+- Qiskit Machine Learning supports the most recent major release with new features, which will only appear in minor releases of that series.
+- The most recent minor release in the current major release series is also supported with bug fixes.
+- The last minor release of the *previous* major release series is supported with bug fixes for six months after a new major release.
+
+For example, if the most recent release is `1.0.1`, then the current major release series is `1.x` and the current minor release is `1.0.x`.
+The `1.0.x` series will be supported with bug fixes, until the release of `1.1.0`, which will include new features.
+The last version of the previous major release, `0.46.x`, is supported with bug fixes only until six months after the final release of `1.0.0`.
+
+As an additional resource, you can find more details on the release and support schedule of Qiskit in the [documentation](https://docs.quantum.ibm.com/start/install#release-schedule).
+
+## Reporting a Vulnerability
+
+To report vulnerabilities, you can privately report a potential security issue
+via the GitHub security vulnerabilities feature. This can be done here:
+
+https://github.com/Qiskit/qiskit/security/advisories
```

Comment:
This should a pointer here not to Qiskit's repo for where to report things. If its in Qiskit they should report it there but ML ones I think not.

Reply:
Indeed it should be https://github.com/qiskit-community/qiskit-machine-learning/security/advisories, just updated it.